free-2.md

```diff
@@ -58,12 +152,37 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+The aforementioned APIs could be used as the backend for the language, so most
+of the semantic work would be in translating DSL commands to API calls. While
+this would certainly be nontrivial, the APIs would eliminate the need to have a
+great deal of technical domain knowledge, so that a CS 111 student could
+probably manage at least a prototype of this language.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+I think this project is definitely on the more ambitious side. Implementing the
+full language with all of the features I have described would be difficult.
+However, a prototype with a limited feature-set would be very feasible for a
+half-semester's work. For example, the prototype could have features for simple
+market and limit orders, and perhaps leave out the more complex stock-selection
+logic. I think this would still be a useful exercise, and if the language were
+designed so that the additional features could easily be added in at a later
+date, it may even serve as a basis for future work, in or out of class.
```

Comment:
I also agree that this can be a good idea for our half semester. It can be a great start to a DSL that impacts many peoples' lives.